test_scripts/Resumption/InteriorVehicleData/OnHashChange/001_OnHashChange_by_module_subscription_without_moduleId.lua

```diff
@@ -0,0 +1,43 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: SDL sends OnHashChange notification after successful subscription to interior vehicle data
+--  with default moduleId
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is not subscribed to modules
```

Comment:
@GetmanetsIrina Mobile application should not be subscribed to all modules. It should be subscribed on one of modules which is tested during this check. 
Please correct the precondition step's description.
This comment relates to other scripts of the PR too.